System-Testing-Deployments-and-Upgrades/Managing-User-Authorization/index.rst

```diff
@@ -0,0 +1,24 @@
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *none*
+
+.. _Managing-User-Authorization:
+
+###########################
+Managing User Authorization
+###########################
+
+.. note::
+
+    Managing user authorization through LOVE is still not fully implemented and tested.
+    This page will be updated once that is available.
+
+Observing specialists and other observatory system administrators-level personnel are in charge of managing :ref:`user authorization <User-Authorization>` to directly command CSCs when executing activities from :ref:`nublado <Control-System-User-Interfaces-Nublado>`.
```

Comment:
system administrators-level personnel?
So this sounds like IT... but I don't think that's who you mean?
Also, it should be "administrator-level"

Reply:
yeah.. not IT but people with admin-level authority. Not sure how to rephrase that though... Updated "administrator-level".

Reply:
that's good enough I think. We've not really defined how that... and it seems a long ways off.